databricks/koalas/groupby.py

```diff
@@ -1105,7 +1105,11 @@ def apply(self, func, *args, **kwargs) -> Union[DataFrame, Series]:
         1    52
         Name: B, dtype: int64
         """
-        from pandas.core.base import SelectionMixin
+        try:
+            from pandas.core.base import SelectionMixin
+            _builtin_table = SelectionMixin._builtin_table
```

Comment:
Yeah,I think we can fix this in. Can we port this change here just for consistency?

https://github.com/apache/spark/pull/33614

Reply:
Added a PR on pyspark for this change :
https://github.com/apache/spark/pull/33687